modules/actors/actor-sheet-ffg.js

```diff
@@ -796,6 +796,7 @@ export class ActorSheetFFG extends ActorSheet {
               const skill = raw_weapons[i].system.skill.value;
               let pool = new DicePoolFFG({'difficulty': 2});
               pool = get_dice_pool(crew_id, skill, pool);
+              pool = await DiceHelpers.getModifiers(pool, raw_weapons[i]);
```

Comment:
Just as a thought exercise (not something to actually do in this PR), these three lines could be rewritten in a number of ways for clarity, cleanness or brevity:

As a single line, with immutability:
```js
const pool = await DiceHelpers.getModifiers(get_dice_pool(crew_id, skill, new DicePoolFFG({'difficulty': 2}));
, raw_weapons[i]);
```
Bit ugly on the eyes and not very clear what's going on. Shortest code is not necessarily cleanest code.

Slightly nicer to read, still promotes the use of immutability where it makes sense:
```js
const initialPool = new DicePoolFFG({'difficulty': 2});
const pool = await DiceHelpers.getModifiers(get_dice_pool(crew_id, skill, initialPool), raw_weapons[i]);
```

or
```js
const initialPool = get_dice_pool(crew_id, skill, new DicePoolFFG({'difficulty': 2}));
const pool = await DiceHelpers.getModifiers(initialPool, raw_weapons[i]);
```

Failing that we could rework some of the helper methods to be more generic.

Reply:
I like it, thanks.